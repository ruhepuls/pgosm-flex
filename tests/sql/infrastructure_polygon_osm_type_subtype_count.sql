SELECT osm_type COLLATE "C", osm_subtype COLLATE "C", COUNT(*)
    FROM osm.infrastructure_polygon
    GROUP BY osm_type COLLATE "C", osm_subtype COLLATE "C"
    ORDER BY osm_type COLLATE "C", osm_subtype COLLATE "C"
;