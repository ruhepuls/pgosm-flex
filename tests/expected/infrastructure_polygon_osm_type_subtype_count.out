aeroway|aerodrome|2
aeroway|apron|1
aeroway|hangar|2
aeroway|helipad|12
aeroway|heliport|2
aeroway|runway|3
emergency|yes|11
power|generator|9
power|plant|1
power|substation|15
tower||1
